prerequisites.md

```diff
@@ -1,88 +1,51 @@
-# Prerequisites to the Tenant-in-a-day-training
+# Prerequisites to the Tenant-in-one-day-training
 
 ## General information
 
-This guide aims to ensure the training proceeds as desired, with all participants having fullfilled all basic pre-requisites.
-Steps everybody needs to perform:
+This guide aims to ensure the Tenant-in-one-Day-training proceeds as desired, with all participants having fullfilled all basic pre-requisites.
 
-1. You will generate your own unique [keypair](##glossary).
-2. You will send your public key to your trainer.
-3. Your public key will be given to your [VM](##glossary), which will then be able to recognize and authenticate your [SSH](##glossary) session.
-4. You will download [UMP](##glossary), to be able to interact with the [DSH](##glossary).
+Required:
 
-Optionally, if you don't already have it, you may need to set up a way to use SSH.
+- Basic knowledge of [SSH](#glossary)
+- A shell capable of SSH, or an ssh client
 
-If you are already familiar with SSH and creating a keypair, you can skip straight to your preferred version of 'connecting'.
+Optional:
 
-## Options to connect over SSH
-
-- Linux and Mac have an SSH client built in. [Proceed to Generating a keypair in Linux/Mac](#linmac).
-- Windows sometimes comes with ssh. To check, you open Powershell (WIN+X, A, click `yes`) and type `ssh`. You will either get feedback stating:
-  - you didn't supply required parameters (meaning you have it, and should proceed to [generating a keypair in PowerShell](#powershellkeygen).
-  - ssh is unknown, meaning you don't have it, so you should continue to the next step.
-- If Windows doesn't have ssh, you have several options:
-  - [Windows Subsystem for Linux (WSL)](#getwsl) allows you to run Linux commands (including SSH) on your Windows system.
-  - [Git for windows](#getgit) allows you to run git commands, through a bash shell. There is even a portable version that does not require admin rights.
-  - Many more options. You can use whatever you're comfortable with. _(note that the format in which PuttyGen stores the keys is _not_ standard; you will need to copy the generated key in a new text file)_
-
-WSL has our preference, with Git for Windows acting as a backup-option. Guide for both options have been included.
+- [UMP](#glossary), to be able to interact with the [DSH](#glossary)
 
-<a name="powershellkeygen"></a>
+Before the training starts, you will receive:
 
-## Generating a keypair in Windows PowerShell
+- The IP-address of your [virtual machine (VM)](#glossary), which will be created for the purposes of this training
+- A private SSH key. This key is part of a [keypair](#glossary), with the other part being given to your VM
 
-1. In PowerShell, type the following commands (press ENTER after each command)
+If you are already familiar with SSH, have a working shell or client, then you can skip straight to your preferred version of 'connecting'.
```

Comment:
... with SSH, *and* have a working shell or client...

Sentence seems weird to me without the 'and'

Reply:
Also, why is 'connecting' between quotes?

Reply:
Restructured the sentence, and made the links explicit.